maskrcnn_benchmark/config/defaults.py

```diff
@@ -272,3 +272,6 @@
 _C.OUTPUT_DIR = "."
 
 _C.PATHS_CATALOG = os.path.join(os.path.dirname(__file__), "paths_catalog.py")
+
+_C.TENSORBOARD_LOGDIR = "logs"
+_C.TENSORBOARD_EXP_NAME = "maskrcnn-benchmark"  # tensorboard experiment name
```

Comment:
Do we actually need those two config values? Can't we instead just use `OUTPUT_DIR` for the TensorBoard logger?